src/document/utils/fill-builder.js

```diff
@@ -0,0 +1,30 @@
+import { fill } from "../builders.js";
+
+function fillBuilder() {
```

Comment:
It might be better to encourage using `fillBuilder()` instead of `fill()` for whole the repository since this should be safer.
If you agree, let's discuss on another discussion or issue.